### PR TITLE
Exersices 1

### PR DESCRIPTION
**Pull request by Andkleven**
Have done every task on exercise 1

### CODE REVIEW

File: src/anders_mathiesen_ex/ex01/comp_to_loop.py
Comment (danielambrosius) [inline src/anders_mathiesen_ex/ex01/comp_to_loop.py]: Avoid print statements outside the main section of the program.

File: src/anders_mathiesen_ex/ex01/deck_by_comp.py
Comment (danielambrosius) [inline src/anders_mathiesen_ex/ex01/deck_by_comp.py]: no blank line at end of file (PEP8 violation)

File: src/anders_mathiesen_ex/ex01/letter_counts.py
Comment (danielambrosius) [inline src/anders_mathiesen_ex/ex01/letter_counts.py]: no blank line at end of file (PEP8 violation)

File: src/anders_mathiesen_ex/ex01/letter_counts.py
Comment (danielambrosius) [inline src/anders_mathiesen_ex/ex01/letter_counts.py:6]: this is not case-insensitive, i.e. 
`In [2]: letter_freq("HHhhhh")                                                      
Out[2]: {'H': 2, 'h': 4}`

File: src/anders_mathiesen_ex/ex01/tidy_code.py
Comment (danielambrosius) [inline src/anders_mathiesen_ex/ex01/tidy_code.py:9]: please write all code in english. Also i think this function definition is a bit too explicit. could be something like `user_guess()`

File: src/anders_mathiesen_ex/ex01/tidy_code.py
Comment (danielambrosius) [inline src/anders_mathiesen_ex/ex01/tidy_code.py]: This is pretty cool, though be wary of using a "bare" except statement as it is usually supposed to catch errors. A better option in this case would be an if-test.

File: src/anders_mathiesen_ex/ex01/tidy_code.py
Comment (danielambrosius) [inline src/anders_mathiesen_ex/ex01/tidy_code.py:21]: roll_dice()?

File: src/anders_mathiesen_ex/ex01/tidy_code.py
Comment (danielambrosius) [inline src/anders_mathiesen_ex/ex01/tidy_code.py]: conventions is to write func names that return booleans as a "questions". i.e. just is_correct. In this case though, `is_equal()` would be much better, as that is what the function does irrespective of the rest of the script.

**Review by danielambrosius — CHANGES_REQUESTED** [review]
Code looks all right except the letter frequencies (see comment). And i would try to not have overly explicit function/variable names.

Unfortunately your code fails PEP8 (see below), so i cannot approve it. You should be fully able to catch this in pycharm looking at the check-mark in the upper right corner before you submit the code.

I would like you to address the PEP8 violations before merging.
EDIT: I have noticed that the acceptance criteria might not have been entirely clear regarding PEP8, so please don't close this pull request until **this thursday**, when Yngve, the other TA's and me have talked it over.

**Comment by danielambrosius** [comment]
Hey!
Unfortunately your hand-in still won't pass. PEP8 quality control has been informed about in [this youtube video](https://www.youtube.com/watch?v=AyINGuxFpnU), and  the task discritpion:

> Check that your code passes the “quality control” in PyCharm!

Be sure to read through the task description before handing in the code as it sucks to fail on formalities.

**Comment by Andkleven** [comment]
d

**Pull request merged**
